indico/web/client/js/utils/case.js

```diff
@@ -35,3 +35,21 @@ export default function camelizeKeys(obj) {
         }
     }, {});
 }
+
+export function snakifyKeys(obj) {
```

Comment:
Maybe it would be better if you could make those functions a bit more generic to minimize code duplication.

Reply:
I tried that but we decided it's better to stay that way.